pkg/tunneler/integration_test.go

```diff
@@ -201,16 +211,19 @@ func Test_integration_listener_reconnect(t *testing.T) {
 	go server2.Serve(l2)
 	defer server2.Close()
 
-	resp, err = client.Get(uri) //nolint:noctx
-	if err != nil {
-		t.Fatalf("Request Failed: %s", err)
-	}
-	defer resp.Body.Close()
+	rw2 := httptest.NewRecorder()
+	b2 := &bytes.Buffer{}
+	req2, err := http.NewRequest(http.MethodGet, dstURL+"/"+proxySubresourcePath+"/", b2) //nolint:noctx
```

Comment:
Why use the `proxySubresourcePath` here ? Does it still make sense ?

Reply:
done